### DISCUSSION TITLE
Amiga performance - recommended tweaks

### DISCUSSION THREAD

**Discussion by AndiTails** [eduardofilo/RG350_adam_image#88] [Ideas & Requests]
Hi,

The default configuration for the Amiga emulation is too slow for RG350-based devices.
The following changes make it run great:

Model - A600 (it defaults to A1200 - the RG350 can't handle that!)
Media -> WHDLoad Splash Screen - both (this allows you to use cheats built into the WHD versions of games)
Video -> Allow PAL/NTSC Hz change (PAL should be set for correct speed, but this should allow NTSC-only games to run too)
Video -> Standard - Pal auto
Video -> Zoom Mode - small
Video -> Frameskip - 2

This, along with adding .lha to the supported filetypes (as #16) has made Amiga games so playable on the RG350! It's great.

Can I also suggest, under Controls, Port 1, I've set Y button to be Up. This allows for games where Up is jump or accellorate, it's far easier to press the Y button than use a d-pad.

Fantastic setup though - thank you so much.

**Comment by retrogamecorps** [comment]
Confirmed, this is a nice setup, thanks @AndiTails.

**Comment by eduardofilo** [comment]
@AndiTails This is nice. I will include your suggestions in next release without doubt. Thank you so much.

**Comment by eduardofilo** [comment]
Included in v1.2
